packages/create-remix/templates/fly-stack/prisma/schema.prisma

```diff
@@ -11,13 +11,29 @@ model User {
   id       String @id @default(cuid())
   email    String @unique
   password String
-  notes    Note[]
+
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+
+  userData UserData?
+}
+
+model UserData {
```

Comment:
Let's call this one "User" and call the other table "Auth"

@ryanflorence, is there a more common convention for this? I'm not a fan of "UserData" as a table name in a database 😅

Reply:
same really, almost went with `Account` but i couldn't decide which one was the account and what was the user haha

Reply:
What about just storing the data on the user model directly and then omitting the password from the returned object by createUser?